stylesheets/components/nav.css

```diff
@@ -0,0 +1,44 @@
+@import url(/Stylesheets/main.css);
+
+* {
+  list-style: none;
+  text-decoration: none;
+}
+
+.navbar {
+  height: 80px;
+  width: 100%;
+  background: var(--navbar-bg);
+}
+
+.logo {
+  position: relative;
+  left: 40px;
+  top: 20px;
+  font-size: xx-large;
```

Comment:
just a suggestion, not mandatory change xx-large could be xx-lg is short and easy to understand

Reply:
lol. bro how can i change that its inbuilt code already